### ISSUE TITLE
Quickstart install Istio problems (PodDisruptionBudget)

### ISSUE THREAD

**Issue by tamis-laan** [kserve/kserve#2477]
I'm trying to use the quickstart install guide and I'm running into `istio` issues:
```
> curl -s "https://raw.githubusercontent.com/kserve/kserve/release-0.9/hack/quick_install.sh" | bash

Flag --short has been deprecated, and will be removed in the future. The --short output will become the default.
bash: line 40: [: v4: integer expression expected
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   101  100   101    0     0   1725      0 --:--:-- --:--:-- --:--:--  1771
100  4856  100  4856    0     0  22349      0 --:--:-- --:--:-- --:--:-- 22349

Downloading istio-1.14.0 from https://github.com/istio/istio/releases/download/1.14.0/istio-1.14.0-linux-amd64.tar.gz ...

Istio 1.14.0 Download Complete!

Istio has been successfully downloaded into the istio-1.14.0 folder on your system.

Next Steps:
See https://istio.io/latest/docs/setup/install/ to add Istio to your Kubernetes cluster.

To configure the istioctl client tool for your workstation,
add the /home/tux/dev/kserve-test/istio-1.14.0/bin directory to your environment path variable with:
	 export PATH="$PATH:/home/tux/dev/kserve-test/istio-1.14.0/bin"

Begin the Istio pre-installation check by running:
	 istioctl x precheck

Need more information? Visit https://istio.io/latest/docs/setup/install/
namespace/istio-system created
✔ Istio core installed
- Processing resources for Istiod.                                                                                                                                                                         2022-10-19T09:05:47.341894Z	 error	 installer	 failed to update resource with server-side apply for obj PodDisruptionBudget/istio-system/istiod: no matches for kind "PodDisruptionBudget" in version "policy/v1beta1"
✘ Istiod encountered an error: failed to update resource with server-side apply for obj PodDisruptionBudget/istio-system/istiod: no matches for kind "PodDisruptionBudget" in version "policy/v1beta1"
- Processing resources for Ingress gateways.                                                                                                                                                               2022-10-19T09:05:47.426293Z	 error	 installer	 failed to update resource with server-side apply for obj PodDisruptionBudget/istio-system/istio-ingressgateway: no matches for kind "PodDisruptionBudget" in version "policy/v1beta1"
✘ Ingress gateways encountered an error: failed to update resource with server-side apply for obj PodDisruptionBudget/istio-system/istio-ingressgateway: no matches for kind "PodDisruptionBudget" in version "policy/v1beta1"
- Pruning removed resources                                                                                                                                                                                2022-10-19T09:05:47.688725Z	 warn	 installer	 retrieving resources to prune type autoscaling/v2beta1, Kind=HorizontalPodAutoscaler: no matches for kind "HorizontalPodAutoscaler" in version "autoscaling/v2beta1"
2022-10-19T09:05:48.328322Z	warn	installer	retrieving resources to prune type policy/v1beta1, Kind=PodDisruptionBudget: no matches for kind "PodDisruptionBudget" in version "policy/v1beta1"
Error: failed to install manifests: errors occurred during operation
```

**Comment by yuzisun** [comment]
@tamis-laan see https://github.com/kserve/kserve/issues/2474

**Comment by yuzisun** [comment]
@tamis-laan two options here as Kube 1.25 dropped the support for PDB v1beta1:
1. Use Kubernetes version lower than 1.25
2. Use quick install script from master branch which installs Istio 1.15 that fixes PDB version issue with Kube 1.25.
